src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    public static final String COMMA = ",";
+    public static final int ZERO_POSITION = 0;
+    public static final int FIRST_POSITION = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File readFromFile = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(readFromFile))) {
+            for (String line = bufferedReader.readLine();
+                    line != null;
+                    line = bufferedReader.readLine()) {
+                if (line.split(COMMA)[ZERO_POSITION].equals(SUPPLY)) {
+                    supply += Integer.parseInt(line.split(COMMA)[FIRST_POSITION]);
+                }
+                if (line.split(COMMA)[ZERO_POSITION].equals(BUY)) {
+                    buy += Integer.parseInt(line.split(COMMA)[FIRST_POSITION]);
+                }
+            }
+            writeToFile(toFileName, supply, buy);
```

Comment:
you open connection ot a file inside another connection to file (try block inside try block). It's considered to be a bad practice. We need to close current resources before opening new one

Reply:
but i use try with resources

Reply:
yes, and at what point will resources be closed?